crates/xilem_masonry/src/view/flex.rs

```diff
@@ -35,7 +44,7 @@ where
         let mut scratch = Vec::new();
         let mut splice = VecSplice::new(&mut elements, &mut scratch);
         let seq_state = self.sequence.build(cx, &mut splice);
-        let mut view = widget::Flex::column();
+        let mut view = widget::Flex::for_axis(self.axis);
```

Comment:
I didn't expose this intentionally, because Masonry doesn't support changing the axis at runtime.

We do not want to have views which give different final trees in `build` and `rebuild`

Reply:
Oh, haven't checked that closely yet. That's unfortunate, and I agree, all attributes that are able to set in the view  should of course change the widget tree.
In that case maybe stick to something like `h_flex` or `h_stack` something a like (open for naming suggestions)?

Reply:
Or, well just implement it in masonry, checking that now...

Reply:
So I've implemented it in masonry, and checked it via

```rust
let axis = if count % 2 == 0 {
    Axis::Horizontal
} else {
    Axis::Vertical
};

...
flex(()).direction(axis)
```

(but have hardcoded the values in the example)